templates/16-04/index.html

```diff
@@ -269,7 +245,7 @@ <h3 class="p-heading--4">Upgrading to Ubuntu 18.04 LTS, Ubuntu 20.04 LTS</h3>
     </div>
     <div class="col-6">
       <h3 class="p-heading--4">Upgrading Ubuntu 16.04 LTS on Azure, AWS</h3>
-      <p>If you are looking to upgrade your Ubuntu 16.04 LTS public cloud instance, it is recommended to launch new, Ubuntu Pro images. Ubuntu Pro for <a href="/azure/pro">Azure</a> and <a href="/aws/pro">AWS</a> are premium images with security and compliance features built in, and optimised for the public cloud.</p>
+      <p>If you are looking to upgrade your Ubuntu 16.04 LTS public cloud instance, it is recommended to launch new, Ubuntu Pro images. Ubuntu Pro for <a href="/azure/pro">Azure</a> and <a href="/aws/pro">AWS</a> are premium images with security and compliance features built-in, and optimised for the public cloud.</p>
       <p>Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. Ideal for companies that have embraced open source for their new products.</p>
```

Comment:
Hey @sowasred2012 this needs to be updated to open-source as well?

Reply:
I thought this at first too, but according to [our copy style guide](https://docs.google.com/document/d/1AX-kSNztuAmShEoohe8L3LNLRnSKF7I0qkZGNeoGOok/edit#heading=h.ylqfftorfys4), "open source" should only be hyphenated when it's an adjective followed by a noun, so not in this case.